src/App/App.js

```diff
@@ -85,24 +85,27 @@ class App extends Component {
 					/> 
 					<Route 
 						exact
-						path='/movies/:id'
-						render={() => {
+						path='/:id'
+						render={({ match }) => {
```

Comment:
Let's ditch 'match' on this Route to single movie, based on our conversation with instructors today. Since we already have our movie object in state and don't need to find the id in an array.